apps/src/gamelab/GameLab.js

```diff
@@ -235,20 +235,45 @@ GameLab.prototype.init = function (config) {
 GameLab.prototype.setupReduxSubscribers = function (store) {
   var state = {};
   var boundOnIsRunningChange = this.onIsRunningChange.bind(this);
+  var boundOnSelectedAnimationChange = this.onSelectedAnimationChange.bind(this);
   store.subscribe(function () {
     var lastState = state;
     state = store.getState();
 
     if (!lastState.runState || state.runState.isRunning !== lastState.runState.isRunning) {
       boundOnIsRunningChange(state.runState.isRunning);
     }
+
+    if (!lastState.animationTab || state.animationTab.selectedAnimation !== lastState.animationTab.selectedAnimation) {
+      boundOnSelectedAnimationChange(state.animationTab.selectedAnimation);
+    }
   });
 };
 
 GameLab.prototype.onIsRunningChange = function () {
   this.setCrosshairCursorForPlaySpace();
 };
 
+GameLab.prototype.onSelectedAnimationChange = function (selectedAnimation) {
+  if (!selectedAnimation) {
+    return;
+  }
+
+  const iframe = document.getElementById('piskel-frame');
+  if (!iframe) {
+    return;
+  }
+  const iframeWindow = iframe.contentWindow;
+  if (!iframeWindow) {
+    return;
```

Comment:
Hmm... I'm being overlay paranoid because this is sort of an event handler, and I'm concerned that it might get called before the initial render.  I suppose we should make sure our initialization doesn't allow that, though.